react/features/screenshot-capture/functions.js

```diff
@@ -18,3 +19,26 @@ export function createScreenshotCaptureSummary(stateful: Object | Function) {
 
     return new ScreenshotCaptureSummary(toState(stateful));
 }
+
+/**
+ * Get a participant's connection JID given its ID.
+ *
+ * @param {Object} state - The redux store state.
+ * @param {string} participantId - ID of the given participant.
+ * @returns {string|undefined} - The participant connection JID if found.
+ */
+export function getParticipantJid(state: Object, participantId: string) {
+    const conference = getCurrentConference(state);
```

Comment:
nit
```
getCurrentConference(state)?
 .getParticipantById(participantId)?
 .getJid()
```

Reply:
"Flow does not yet support method or property calls in optional chains."